ompi/dpm/dpm.c

```diff
@@ -1974,6 +1975,46 @@ static void set_handler_default(int sig)
     sigaction(sig, &act, (struct sigaction *)0);
 }
 
+static char *find_prte(void)
```

Comment:
We already have a function doing the exact same thing in `ompi/tools/mpirun/main.c`. Instead of duplicating code maybe we should unify the two and have a consistent approach.

Reply:
I will look forward to your PR. This seemed a rather trivial duplication for a very limited purpose.